packages/notebook/src/actions.tsx

```diff
@@ -44,12 +74,40 @@ const JUPYTER_CELL_MIME = 'application/vnd.jupyter.cells';
  */
 export class NotebookActions {
   /**
-   * A signal that emits whenever a cell is run.
+   * A signal that emits whenever a cell completes execution.
    */
-  static get executed(): ISignal<any, { notebook: Notebook; cell: Cell }> {
+  static get executed(): ISignal<
+    any,
+    {
+      notebook: Notebook;
+      cell: Cell;
+      success: boolean;
+      error?: KernelError | null;
+    }
+  > {
     return Private.executed;
   }
 
+  /**
+   * A signal that emits whenever a cell execution is scheduled.
+   */
+  static get executionScheduled(): ISignal<
+    any,
+    { notebook: Notebook; cell: Cell }
+  > {
+    return Private.executionScheduled;
+  }
+
+  /**
+   * A signal that emits whenever a cell completes execution.
```

Comment:
This docstring needs updating (copy-paste error). The docstring in implementation is fine.

Reply:
I opened https://github.com/jupyterlab/jupyterlab/pull/10579